spec/models/miq_policy_content_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe MiqPolicyContent do
+  describe ".seed" do
+    it "should contain conditions" do
+      [MiqAction, MiqEventDefinition, MiqPolicy, MiqPolicyContent].each(&:seed)
+
+      specifications = YAML.load_file(File.join(ApplicationRecord::FIXTURE_DIR, "miq_policy_contents.yml"))
+      specifications_by_pkey = specifications.index_by do |h|
+        "#{h[:miq_policy]}_#{h[:success_sequence]}_#{h[:failure_sequence]}"
+      end
+
+      count = 0
+      MiqPolicyContent.all.each do |mpc|
+        spec = specifications_by_pkey["#{mpc.miq_policy.name}_#{mpc.success_sequence}_#{mpc.failure_sequence}"]
+        expect(mpc).to have_attributes(
+          spec.except(:created_on, :updated_on, :miq_policy, :miq_action, :miq_event_definition)
```

Comment:
I don't know if this list will grow, but it might be a pain to maintain,  keep it in mind if we have to alter this list of columns.

I think we could test one or two attributes we care about instead of checking all attributes not in the this exclude list.